config.json

```diff
@@ -39,6 +39,7 @@
     "leap",
     "space-age",
     "pascals-triangle",
+    "sieve",
```

Comment:
I moved this a little bit further down to give people more time to get familiar with Lua before tackling coroutines.